core-types/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "core-types"
```

Comment:
Please go with `bounded-collections` or whatever was the name before. This is just tooo generic.